src/main/java/baritone/process/CraftingProcess.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * This file is part of Baritone.
+ *
+ * Baritone is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Baritone is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Baritone.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package baritone.process;
+
+import baritone.Baritone;
+import baritone.api.process.ICraftingProcess;
+import baritone.api.process.PathingCommand;
+import baritone.api.process.PathingCommandType;
+import baritone.utils.BaritoneProcessHelper;
+import net.minecraft.client.gui.GuiScreen;
+import net.minecraft.client.gui.inventory.GuiCrafting;
+import net.minecraft.client.util.RecipeItemHelper;
+import net.minecraft.init.Blocks;
+import net.minecraft.inventory.ClickType;
+import net.minecraft.inventory.ContainerWorkbench;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.crafting.CraftingManager;
+import net.minecraft.item.crafting.IRecipe;
+
+import java.util.ArrayList;
+
+public final class CraftingProcess extends BaritoneProcessHelper implements ICraftingProcess {
+
+    //todo this is a copy paste of the getToBlockProcess and hasnt been cleand up
+
+    private Item item;
+    private int amount;
+
+    public CraftingProcess(Baritone baritone) {
+        super(baritone);
+    }
+
+
+    @Override
+    public boolean isActive() {
+        return item != null && amount >= 1;
+    }
+
+    @Override
+    public synchronized PathingCommand onTick(boolean calcFailed, boolean isSafeToCancel) {
+        if (baritone.getGetToBlockProcess().isActive()) {
+            //we are pathing to a table and therefor have to wait.
+            return new PathingCommand(null, PathingCommandType.DEFER);
```

Comment:
No, please not again. No inter process delegation.

I know it's tempting and mostly works, but there's details which make it awful.
Unverified example off the top of my head: If `CraftingProcess` is running and delegating to `GetToBlockProcess` and the user enters another command (e.g. `#farm`) this will cancel `CraftingProcess` as expected *unless* they used a command which uses `GetToBlockProcess` (e.g. `#goto chest`). In that case Baritone will walk to the block as expected, but once it's there `CraftingProcess` will take control and think it's arrived at a crafting table.
The fact that you made `GetToBlockProcess` temporary while `CraftingProcess` is active probably means you already found another unfavorable interaction (and hackpatched over it).